### DISCUSSION TITLE
A tool for fuzzing the right payload

### DISCUSSION THREAD

**Discussion by nirzaaa** [frohoff/ysoserial#224] [Show and tell]
Hi guys!

While working on a PortSwigger lab, I had the problem of finding which payload should work in my case.
For this scenario I created this tool that helps to do the job for us automatically:
https://github.com/nirzaaa/LookingSerial

Video explanation about the tool:
https://www.youtube.com/watch?v=NAK_a324dco&list=PLGj3IZkhXGzIeWYBbXG7P7YEhR8qUSjiH&index=1

Video explanation about modifying the tool for specific cases:
https://www.youtube.com/watch?v=mazIz0Tksjs&list=PLGj3IZkhXGzIeWYBbXG7P7YEhR8qUSjiH&index=2

Your are more than welcome to take a look and leave a comment with your feedback and such,

Have a good one,

nirza

**Comment by PoMa2002** [comment]
can u help me with labs?

**Reply by nirzaaa** [reply]
Sure! Feel free to let me know :)
Will also be able to modify the tool as needed ;)

**Reply by PoMa2002** [reply]
maybe you have discord?

**Comment by nirzaaa** [comment]
> maybe you have discord?

Hi :)

The idea will be to create a discord server where people will be able to ask questions and learn from others.
Until then, feel free to take a look at my YouTube channel here: https://www.youtube.com/channel/UCgAdCrhOYKQrft3uF8yGHOg
I tried to upload videos regarding topics that lack proper explanation in my opinion.

Have a good one,

nirza
